### DISCUSSION TITLE
Course Enrollment & Certification

### DISCUSSION THREAD

**Discussion by aroojfatima7** [giswqs/geog-312#50] [Q&A] [answered]
Hello Professor Qiusheng Wu, 
I've been following your course through YouTube and found it to be very well-designed. I'm interested in getting enrolled in the course but I have a few queries to ask:

1. Can I enroll anytime in the course or is there a specific time period?
2. After enrolling, am I automatically eligible for the certificate, or is there a certain grade percentage required to obtain it?

**Comment by giswqs** [comment, marked as answer]
Thank you for your interest in enrolling in the course. Yes, you can enroll at any time. The certificate will be issue once the 10 lab assingments are submitted and the total grade is great then 60%.

**Reply by aroojfatima7** [reply]
Understood. Thank you very much for your response professor!
